docs/examples/stable_diffusion/README.md

```diff
@@ -0,0 +1,82 @@
+# Stable diffusion
+
+This example focuses on showcasing two of Triton Inference Server's features:
+* Using multiple frameworks in the same inference pipeline
+* Using the Python Backend API to build complex non linear pipelines.
```

Comment:
It would be good to give a brief overview on the use case of stable diffusion (text to image) and what the inference pipeline looks like (what are the models and how they interact with each other).

With the problem statement described, we can have sections ([examples](https://github.com/triton-inference-server/server/tree/main/docs/examples/jetson/concurrency_and_dynamic_batching#demonstration--case-1-concurrent-model-execution)) to highlight what Triton features are provided so that user can integrate such pipeline in Triton, and possibly link to the detailed doc.

For example, in this case, "using the Python Backend API to build complex non linear pipelines" is constructing the pipeline via BLS and multi-frameworks support via Triton backend API and Triton-maintained backends.